bin/run-fiberassign

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python
+
+"""
+Script handling fiberassign inputs
+"""
+
+import sys, os
+import argparse
+import desimodel.io
+import numpy as np
+import datetime
+from desitarget.targetmask import desi_mask
+
+
+parser = argparse.ArgumentParser()
+parser.add_argument("--mtl", type=str,  help="input targets (FITS file)", required=True)
+parser.add_argument("--sky", type=str,  help="input sky positions (FITS file)", required=True)
+parser.add_argument("--stdstar", type=str,  help="input std stars (FITS file)", required=True)
+parser.add_argument("--fibstatusfile", type=str,  help="list of positioners and its status (ECSV file)", required=True)
+parser.add_argument("--footprint", type=str,  help="list of tiles defining the footprint (FITS file)")
+parser.add_argument("--positioners", type=str,  help="list of positioners on the focal plane (text file)")
+
+parser.add_argument("--surveytiles", type=str,  help="set of tiles to run fiberassign on (text file)")
+
+parser.add_argument("--telra", type=float, help="Right Ascension of arbitrary pointing - overrides --surveytiles")
+parser.add_argument("--teldec", type=float, help="Declination of arbitrary pointing - overrides --surveytiles")
+parser.add_argument("--telID", type=int, help="Integer ID of arbitrary pointing - overrides --surveytiles")
```

Comment:
It looks like these options aren't used yet; let's comment them out until they are implemented.

`--telID` should be `--tileid`

Reply:
This is fixed now.